assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -60,7 +60,94 @@ describe( 'modules/analytics accounts', () => {
 	} );
 
 	describe( 'actions', () => {
+		describe( 'createAccount', () => {
+			it( 'creates an account and adds it to the store ', async () => {
```

Comment:
This description isn't correct here, should maybe be something like "creates an account ticket and redirects to the TOS".

Reply:
updated